### ISSUE TITLE
NVDA leaves a table when browsing blank cells in MS Word with UIA enabled

### ISSUE THREAD

**Issue by brunoprietog** [nvaccess/nvda#11043]
With UIA enabled for Microsoft Word, when you navigate a table that contains empty cells, NVDA leaves the table as it passes through that cell, using the Alt+CTRL+Arrow navigation commands. If you navigate only with arrows, for example, with a right arrow to move to that empty cell, it does not report the cell/column/row change.

### Steps to reproduce:

With UIA enabled, insert a table in Microsoft Word and try to navigate it with the table navigation commands, or just the arrows. NVDA does not identify the cells.
If you type things into any of the cells and leave other cells empty, you will be able to move correctly through the cells that have text, but you will not be able to move through the blank cells, NVDA leaves the table or does not announce the cell change.

### Actual behavior:

NVDA does not identify the blank cells, leaves the table or does not announce the cell change.

### Expected behavior:

NVDA announces the cells correctly.

#### NVDA installed/portable/running from source:

Installed and portable

#### NVDA version:

2020.1 beta 1

#### Windows version:

Windows 10Ver1909 (64 bits) build 18363.778

#### Name and version of other software in use when reproducing the issue:

Microsoft Office 16.0.12624.20466

#### Does the issue still occur after restarting your computer?

Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

I get the same behavior with NVDA 2019.3.1 and 2019.2.1

#### If addons are disabled, is your problem still occuring?

Yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?

Yes

**Comment by fisher729** [comment]
Hi.

I can confirm that this issues occurs.

**Comment by brunoprietog** [comment]
@leonardder @feerrenrut @michaelDCurran Friends, have you been able to investigate this error? It is being very critical for many people. Thank you very much in advance.
